docker.go

```diff
@@ -30,10 +30,12 @@ type Container struct {
 
 // PortMapping is the portmapping structure between container and mesos
 type PortMapping struct {
-	ContainerPort int    `json:"containerPort,omitempty"`
-	HostPort      int    `json:"hostPort"`
-	ServicePort   int    `json:"servicePort,omitempty"`
-	Protocol      string `json:"protocol,omitempty"`
+	ContainerPort int                `json:"containerPort,omitempty"`
+	HostPort      int                `json:"hostPort"`
+	Labels        *map[string]string `json:"labels,omitempty"`
```

Comment:
We normally provide convenience methods to easily populate pointerized variables. For labels, we already have something to [add](https://github.com/gambol99/go-marathon/blob/63fea16e82279c1d934bcb4602614308b69f22c3/application.go#L238) and [clear labels](https://github.com/gambol99/go-marathon/blob/63fea16e82279c1d934bcb4602614308b69f22c3/application.go#L250). I think we should duplicate these functions on `PortMapping`.

(I initially thought we could provide a single implementation on a custom `map[string]string` type but came to believe this won't work due to the fact that we can't initialize a receiver map on a receiver method.)